### DISCUSSION TITLE
Duda sobre el registro para el hackaton

### DISCUSSION THREAD

**Discussion by DaviiidSG** [Launch-X-Latam/MisionFrontEnd#316] [Q&A]
Hola compañeros, tengo una duda acerca del registro para el hackaton que enviaron por teams, en los requisitos colocados en el mensaje indica que debemos tener una cuenta de estudio y luego en el form de registro te pide el nombre de la institución donde estudias, pero qué haría en caso de estudiar 100% autodidacta? Cómo podría obtener la cuenta de estudios de github y qué debería colocar en el form?

**Comment by DiegoCorchuelo** [comment]
Tengo la misma duda, si alguien pudiera ayudarnos sería genial.

**Reply by AlvaroFierro** [reply]
Considero que es innovaccion
[Como obtener la cuenta de estudiante](https://github.com/Launch-X-Latam/MisionFrontEnd/discussions/25 )

**Reply by DiegoCorchuelo** [reply]
Luego de ese paso dice que tenemos que tomar foto de un documento o credencial que verifique que estamos ahora mismo vinculados a esa institución.

**Reply by AlvaroFierro** [reply]
> Luego de ese paso dice que tenemos que tomar foto de un documento o credencial que verifique que estamos ahora mismo vinculados a esa institución.

Checa bien el link que puse, ahi vienen todos los pasos

**Comment by AlvaroFierro** [comment]
También, sabe alguien bien que horario se va a estar usando si el que se comunico via Teams o via el sitio web
![image](https://user-images.githubusercontent.com/71525061/199880454-58d756b0-cbe4-45e7-a7dd-626e64b67b74.png)
![image](https://user-images.githubusercontent.com/71525061/199880478-68caafca-7306-47f3-a5e9-bd3cd51efb14.png)
